From 6821edb2d40956e5f93a80420ae1dbf825b95281 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Mon, 11 Mar 2013 13:26:57 -0400
Subject: [PATCH] fix DESTDIR handling with udev rules install again

Commit 102e29d16c3fd9b414eeac53c794302f902ae8da reverted the DESTDIR
logic when upgrading the udev path handling.

I fixed that with commit a41cd69c87e312198834a2e6232134176c9352e5.

It's been reverted yet again in an unrelated commit
57e9bdeae2f00664616ee1d9a936a5cfe27bfd30.  Why do people keep doing this?

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 SConstruct | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/SConstruct b/SConstruct
index 51b07e1..caed21e 100644
--- a/SConstruct
+++ b/SConstruct
@@ -1790,10 +1790,10 @@ if env['python']:
 # is plugged in.
 
 Utility('udev-install', 'install', [
-    'mkdir -p ' + env['udevdir'],
-    'cp $SRCDIR/gpsd.rules ' + env['udevdir'] + '/rules.d/25-gpsd.rules',
-    'cp $SRCDIR/gpsd.hotplug ' + env['udevdir'],
-    'chmod a+x ' + env['udevdir'] + '/gpsd.hotplug',
+    'mkdir -p ' + DESTDIR + env['udevdir'] + '/rules.d',
+    'cp $SRCDIR/gpsd.rules ' + DESTDIR + env['udevdir'] + '/rules.d/25-gpsd.rules',
+    'cp $SRCDIR/gpsd.hotplug ' + DESTDIR + env['udevdir'],
+    'chmod a+x ' + DESTDIR + env['udevdir'] + '/gpsd.hotplug',
         ])
 
 Utility('udev-uninstall', '', [
-- 
1.8.4.3

